src/extension.ts

```diff
@@ -290,6 +291,42 @@ export function activate(context: vscode.ExtensionContext) {
     }
   });
 
+  const copyContextCommand =  vscode.commands.registerCommand('path-copy.copyrelative', () => {
+		vscode.window.showInformationMessage('Copied File Path');
+		if(vscode.window.activeTextEditor) {
+			currPath = vscode.window.activeTextEditor.document.uri.fsPath
+		}
+	});
+
+	const copyExplorerCommand =  vscode.commands.registerCommand('path-copy.copyexplorerelative', (uri:vscode.Uri) => {
+		currPath = uri.fsPath;
```

Comment:
Make a single util method to copy the path. This function and the above function does the same thing